docs/serving/revision-gc.md

```diff
@@ -0,0 +1,19 @@
+# Revision Garbage Collection
+
+Knative will automatically clean up inactive revisions as configured by the operator. For more information, see the [operator settings](../admin/serving/revision-gc.md).
+
+To always retain a revision add the annotation `serving.knative.dev/no-gc: "true"`
```

Comment:
```suggestion
You can configure a Service so that at least one revision is always retained, by adding the `serving.knative.dev/no-gc: "true"` annotation:
```

Reply:
Updated the example to be more clear where the annotation is supposed to end up